pkg/urbit/jets/tree.c

```diff
@@ -883,6 +888,10 @@ static u3j_core _141_qua_d[] =
 
   { "mink", 7, _141_qua_mink_a, 0, _141_qua_mink_ha },
   { "mule", 7, _141_qua_mule_a, 0, _141_qua_mule_ha },
+
+  { "scot", 7, _141_qua_scot_a, 0, 0 },
+  { "scow", 7, _141_qua_scow_a, 0, 0 },
+  { "slaw", 7, _141_qua_slaw_a, 0, 0 },
```

Comment:
In keeping with the other new-jet PRs, please stub out the arm hash declarations in this file.